hugo/assets/scss/components/blogs.scss

```diff
@@ -0,0 +1,92 @@
+@import '../config/colors';
+@import '../config/sizes';
+@import '../mixins/container';
+@import '../mixins/list-reset';
+@import '../mixins/stretch';
+
+.blogs {
```

Comment:
we already have a list bem component set up for this. list--overview (from an early design). You can use that bem and move the styles there.